Tasks/PublishSymbols/SymbolsCommon.psm1

```diff
@@ -85,7 +85,18 @@ function Get-SymbolBinaryPath
         [string]$V1AgentPath
     )
 
-    $useDbgLkg = [System.Convert]::ToBoolean($env:PublishSymbols_UseDbgLkg)
+    $useDbgLkg = $env:PublishSymbols_UseDbgLkg
+    if ($env:PublishSymbols_UseDbgLkg -eq $null)
+    {
+        # Opt-in to latest symbol binaries by default
+        $useDbgLkg = $true
+    }
+    else
+    {
+        # Opt-out if customer specified by variable in their build
+        $useDbgLkg = [System.Convert]::ToBoolean($env:PublishSymbols_UseDbgLkg)
```

Comment:
>[System.Convert]::ToBoolean($env:PublishSymbols_UseDbgLkg) [](start = 21, length = 58)

[bool]$var

Reply:
Unless this would modify the behavior to fix a bug, then I prefer low-context syntax over PowerShell's high-context mneumonic.